extension/src/popup/basics/layout/View/styles.scss

```diff
@@ -134,8 +126,8 @@
     margin: 0 auto;
     // 848px = 796 + 16 * 2 (padding) + 20 (scrollbar)
     max-width: pxToRem(848px);
-    padding-left: 1rem;
-    padding-right: 1rem;
+    padding-left: 1.5rem;
+    padding-right: 1.5rem;
```

Comment:
very nit: would you mind using pxToRem here so we can continue replacing all rem instances with pxToRem?

Reply:
Sure thing, will do. Is the idea to use pxToRem in 100% of times? Or just in a few specific cases?

Reply:
https://github.com/stellar/freighter/pull/1706/commits/8932350118c187035d2b103d9c278a65da42ae00

Reply:
Thanks! Yeah, I think it'd be nice to use pxToRem in all cases and hopefully over time, we'll replace all instances of rem with pxToRem for consistency